docroot/themes/contrib/civictheme/civictheme_library/components/00-base/_variables.components.scss

```diff
@@ -470,16 +470,16 @@ $ct-video-ratio-height: 9 !default;
 $ct-attachment-border-radius: $ct-border-radius !default;
 $ct-attachment-space-horizontal: ct-spacing(7) !default;
 $ct-attachment-space-vertical: ct-spacing(10) !default;
-$ct-attachment-light-background-color: ct-color-light('background-light') !default;
-$ct-attachment-light-border-color: ct-color-light('border-light') !default;
+$ct-attachment-light-background-color: ct-color-tint(ct-color-light('background-light'), 10) !default;
```

Comment:
@joshua-salsadigital 
please undo this change - we are trying to run away from custom colours like this.

If there is a custom colour - please comment on the ticket that the design team should update figma